docs/management/connectors/action-types/servicenow-sir.asciidoc

```diff
@@ -1,12 +1,70 @@
 [role="xpack"]
 [[servicenow-sir-action-type]]
-=== ServiceNow connector and action
+=== ServiceNow SecOps connector and action
 ++++
 <titleabbrev>ServiceNow SecOps</titleabbrev>
 ++++
 
 The ServiceNow SecOps connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow security incidents.
 
+[float]
+[[servicenow-sir-connector-prerequisites]]
+==== Prerequisites
+After upgrading to 7.16, you must complete the following within your ServiceNow instance before creating a new ServiceNow SecOps connector or <<servicenow-sir-connector-update, updating an existing one>>:
+
+* Install [Elastic for Security Operations (SecOps)] https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f from the ServiceNow Store.
+* Create a ServiceNow integration user and assign it the appropriate roles.
+* Create a CORS rule.
```

Comment:
Not sure if it's a common concept in Elastic docs, but maybe we should define CORS the first time we use it in this doc. Cross-origin resource sharing?